python_testspace_xml/testspace_xml.py

```diff
@@ -40,8 +40,22 @@ def __init__(self, name='unknown', level='info', description='',
         self.description = description
         self.mimeType = mime_type
         self.file_path = file_path
+        self.gzip_data = None
         self.comments = []
 
+        if self.file_path is not None:
+            if not os.path.isfile(self.file_path):
+                ta = Annotation(self.file_path, level='error')
+                ta.description = 'File: ' + self.file_path + ' not found.'
+                self.comments.append(ta)
```

Comment:
Yes, this needs refactored as it currently doesn't work. I will do in a new branch once this once is merged